pyfar/classes/coordinates.py

```diff
@@ -1456,20 +1878,7 @@ def _make_system(self, domain=None, convention=None, unit=None):
 
         return system
 
-    def _return_system(self, pts1, pts2, pts3, new_system, convert):
```

Comment:
I think the internal _system is not needed in the future, since it is always saved in cartesian.